csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1200,9 +1527,17 @@ The Summary of the revision (`summary`) of value type `string` with 1 or more ch
 Document status (`status`) of value type `string` and `enum` defines the draft status of the document.
 The value MUST be one of the following:
 
+```
     draft
     final
     interim
+```
+
+The value `draft` indicates, that this is a pre-release, intended for issuing party’s internal use only, or possibly used externally when the party is seeking feedback or indicating its intentions regarding a specific issue.
+
+The value `final` indicates, that the issuing party asserts the content is unlikely to change. “Final” status is an indication only, and does not preclude updates.
+
+The value `interim` indicates, that the issuing party asserts the content is unlikely to change.
```

Comment:
@sthagen: This looks like the same definition as `final`. I would suggest the following:
```suggestion
The value `interim` indicates, that the issuing party asserts the content of the current version is unlikely to change. However, further updates of this advisory are already planned.
```

Reply:
@santosomar, @mprpic, @tolim: Are you okay with the change?

Reply:
"Content is unlikely to change" and "advisory is planned to be updated" kind of seem like opposite statements. I'm actually not sure what the purpose of the "interim" status is. Either the advisory is in draft and may change, or it's in its final form and is ready for consumption. As a consumer, I'm not sure I would care about a third state when even the "final" state does not mean the content won't ever change again (that's what revision history is for).

Reply:
I would think of amending an already present content with additional content or measures? But here better practitioners having to produce the incremental versions do speak up.

Reply:
This is now tracked as #226. The solution will be probably part of one of the next editor revisions.